container-engine-lib/lib/backend_impls/docker/docker_kurtosis_backend/docker_kurtosis_backend.go

```diff
@@ -117,6 +117,8 @@ func (backend *DockerKurtosisBackend) CreateEngine(
 	gitAuthToken string,
 	sinks logs_aggregator.Sinks,
 	shouldEnablePersistentVolumeLogsCollection bool,
+	logsCollectorFilters []logs_collector.Filter, // ignored on docker backend for create engine
```

Comment:
Thought going off in my brain: this seems like a thing that's useful to have in user-facing documentation somewhere

Reply:
I should've specified the comment more but its not actually ignored, its only ignored by the `CreateEngine` function, the effect of logsCollectorFilters is still applied by `CreateLogsCollectorForEnclave` - I can specify the comment but from a user perspective, logs collector filters will work over both docker and k8s